includes/wikia/tasks/Tasks/SiteWideMessagesTask.class.php

```diff
@@ -25,6 +31,8 @@ public function init() {
 		if (!defined('MSG_STATUS_UNSEEN')) {
 			define('MSG_STATUS_UNSEEN', '0');
 		}
+
+		$this->userGroupStorage = Injector::getInjector()->get(UserGroupStorage::class);
 	}
 
 	public function send($args) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this function "send" is 32 which is greater than 20 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)